apps/extension/src/ui/apps/dashboard/routes/Settings/Mnemonics/MnemonicBackupModal/ModalRouter.tsx

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable react/display-name */
```

Comment:
Could potentially remove the eslint-disable line and instead add the following at the end of the file:
```ts
MnemonicBackupModalRouter.displayName = "MnemonicBackupModalRouter"
```

Reply:
Thanks, actually I don't think that's even required - was just cruft from previous attempts